components/colorloupe/stories/template.js

```diff
@@ -4,18 +4,202 @@ import { classMap } from 'lit-html/directives/class-map.js';
 
 import '../index.css';
 import '../skin.css';
+import '../../expressvars/custom.css';
 
 export const Template = ({
   rootClass = "spectrum-ColorLoupe",
-  size = "m",
+  isOpen,
+  style = "--spectrum-picked-color: rgba(255, 0, 0, 0.5); position: absolute; top: 25%; left: 50%;",
+  darkColor = "rgb(217, 217, 217)",
+  lightColor = "rgb(255, 255, 255)",
```

Comment:
```suggestion
  darkColor = "rgb(217, 217, 217)",
  lightColor = "rgb(255, 255, 255)",
```

^ you could surface these in the controls as color values if you like

Reply:
was able to access the actual var `--spectrum-colorcontrol-checkerboard-dark-color`  and got these removed